### PR TITLE
Changes for no filter and no results

### PR DESCRIPTION
**Pull request by ShikhaMishra9**

### CODE REVIEW

File: components/ImprovedSearch/Hits.tsx [hunk 1]
```diff
@@ -28,6 +34,18 @@ type Props = {
 export const Hits = () => {
   const { hits } = useHits()
   const { query } = useSearchBox()
+
+  if (!hits || hits.length === 0) {
+    return (
+      <StyledList variant="numbered">
+        <NoRelevant>
```
Comment (wenche) [inline components/ImprovedSearch/Hits.tsx:41]: It's not allowed to add a span inside an ordered list, we need another container than a list to wrap this

File: components/ImprovedSearch/Hits.tsx [hunk 2]
```diff
@@ -28,6 +34,18 @@ type Props = {
...
+          <Typography group="paragraph" variant="body_short" style={{ color: "Tertiary", textAlign: "center" }}>
```
Comment (wenche) [inline components/ImprovedSearch/Hits.tsx:42]: Can't add color as a string `tertiary`

File: components/ImprovedSearch/RefinementList.tsx [hunk 3]
```diff
@@ -53,7 +53,9 @@ export const RefinementList = (props: RefinementListProps) => {
         </EdsProvider>
       ) : (
         <NoRelevant>
-          <FormattedMessage id="asset.no.relevant.filters" defaultMessage="No assets for this filter" />
+          <Typography group="paragraph" variant="meta" color="Tertiary">
```
Comment (wenche) [inline components/ImprovedSearch/RefinementList.tsx:56]: If you inspect using dev tools you will see that `color="tertiary"` doesn't work

**Review by wenche — CHANGES_REQUESTED** [review]
The "Filter" header does still not show up, did you agree with Espen to do that in another PR?

File: components/ImprovedSearch/Hits.tsx (hunk 1 above)
Comment (ShikhaMishra9) [reply components/ImprovedSearch/Hits.tsx:41]: ok 

> The "Filter" header does still not show up, did you agree with Espen to do that in another PR?

yes i thought to have it in another pat after checking with you once

File: components/ImprovedSearch/RefinementList.tsx (hunk 3 above)
Comment (ShikhaMishra9) [reply components/ImprovedSearch/RefinementList.tsx:56]: yeah ... i  kept it to ask ...if this is something we can use through token colors

**Review by wenche — CHANGES_REQUESTED** [review]
Remove the StyledList :thumbsup:

**Review by wenche — APPROVED** [review]
Looks good now 👍

**Pull request merged**